x-pack/plugin/core/src/main/java/org/elasticsearch/license/StartTrialClusterTask.java

```diff
@@ -102,8 +104,8 @@ public ClusterState execute(ClusterState currentState) throws Exception {
     }
 
     @Override
-    public void onFailure(String source, @Nullable Exception e) {
-        logger.error(new ParameterizedMessage("unexpected failure during [{}]", source), e);
+    public void onFailure(@Nullable Exception e) {
+        logger.error(new ParameterizedMessage("unexpected failure during [{}]", TASK_SOURCE), e);
```

Comment:
I guess we could just use String concat here and drop the `ParameterizedMessage` since this would compile into a string literal then anyway.